### PR TITLE
Link to more info about the MANAGED_PROJECT

### PR DESCRIPTION
**Pull request by mnmami**
It was not clear what to put in MANAGED_PROJECT, especially that opening ./instance/managed-project/iam.yaml put the MANAGED_PROJECT as the namespace. More description here is needed.

### CODE REVIEW

**Comment by mnmami** [comment]
@googlebot I signed it!

File: content/en/docs/gke/deploy/management-setup.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ to manage GCP infrastructure using GitOps.
 In the last step we created the GCP service account **${NAME}-cnrm-system@${PROJECT}.iam.gserviceaccount.com**
 this is the service account that CNRM will use to create any GCP resources. Consequently
 you need to grant this GCP service account sufficient privileges to create the desired
-resources in one or more projects. 
+resources in one or more projects (called managed projects, read [more](https://github.com/kubeflow/gcp-blueprints/tree/master/management/instance/managed-project)).
```
Comment (8bitmp3) [inline content/en/docs/gke/deploy/management-setup.md:100]:
```diff
- Consequently
+ Consequently,
you need to grant this GCP service account sufficient privileges to create the desired
- resources in one or more projects (called managed projects, read [more]
- (https://github.com/kubeflow/gcp-blueprints/tree/master/management/instance/managed-project)). 
+ resources in one or more projects. These are called 
+ [managed projects](https://github.com/kubeflow/gcp-blueprints/tree/master/management/instance/managed-project)).
```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
Hey @mnmami Thanks for the short PR!

My only suggestion is to break the sentence into two simpler ones, while shortening the second added thing and keeping the main gist.

```diff
- Consequently
+ Consequently,
you need to grant this GCP service account sufficient privileges to create the desired
- resources in one or more projects (called managed projects, read [more]
- (https://github.com/kubeflow/gcp-blueprints/tree/master/management/instance/managed-project)). 
+ resources in one or more projects. These are called 
+ [managed projects](https://github.com/kubeflow/gcp-blueprints/tree/master/management/instance/managed-project)).
```

What do you think?

cc @Bobgy (he knows way more about deployments than 8bitmp3). I think @Bobgy is also working on something within `management-setup.md`—I've just reviewed one of the PRs.

File: content/en/docs/gke/deploy/management-setup.md (hunk 1 above)
Comment (Bobgy) [inline content/en/docs/gke/deploy/management-setup.md:100]: Thanks @mnmami! It's probably better directly include that paragraph here.

**Comment by joeliedtke** [comment]
Preview: https://deploy-preview-2153--competent-brattain-de2d6d.netlify.app/docs/gke/deploy/management-setup/

**Comment by Bobgy** [comment]
Can you please fix the merge conflict?
/lgtm
otherwise

**Comment by Bobgy** [comment]
@mnmami Thank you for your contribution!
/lgtm
/approve

**Pull request merged**